### DISCUSSION TITLE
Notifications API - Marketplace Id

### DISCUSSION THREAD

**Discussion by damianelder** [amzn/selling-partner-api-models#3104] [General]
https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/notifications-api-use-case-guide/notifications-use-case-guide-v1.md#step-1-grant-selling-partner-api-permission-to-write-to-your-sqs-queue

Step 2. Create a destination (For sqs)
Step 3. Create a subscription (for sqs)

Doesn't seem to be an option to specify a marketplace id like MWS.  is this handled differently?  

![image](https://user-images.githubusercontent.com/64195537/128221227-d3a5c249-3ecf-4637-91c5-05a98769a1bd.png)

**Comment by naocan12345** [comment]
http://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html Data introduction with interface
